app/controllers/concerns/content_items_concern.rb

```diff
@@ -0,0 +1,19 @@
+module ContentItemsConcern
```

Comment:
@ikennaokpala I am happy with extracting a concern if the pattern applies multiple times 👍 , but would you mind keeping this commit for the future (I am not sure about the name, for example) The same sorting pattern might apply to other lists, for example.